### PR TITLE
A0-2411: audit kz azse 10 user experience

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: packages/extension/src/page.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { packageInfo } from './packageInfo';
 
 function inject () {
   injectExtension(enable, {
-    name: 'polkadot-js',
+    name: 'aleph-zero-signer',
```
[the remark this answers is not in the document]
Comment (youPickItUp) [reply packages/extension/src/page.ts:15]: correction, there was a change required in dashboard https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/pull/38

File: packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx [hunk 2]
```diff
@@ -22,15 +31,31 @@ interface Props extends ThemeProps {
...
-    if (success) {
-      show(t('Secret phrase copied to clipboard'), 'success');
-    } else {
-      show(t('Failed copying to clipboard'), 'critical');
-    }
-  }, [show, t]);
+  const _onCopy = useCallback(
+    (test: string, success: boolean) => {
+      if (success) {
+        show(t('Secret phrase copied to clipboard'), 'success');
```
Comment (jalooc) [inline packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx:42]: How about going crazy and:
```suggestion
  const _onCopy = useCallback(
    (test: string, success: boolean) => {
      if (success) {
        setIsSecretCopied(true)
        show(t('Secret phrase copied to clipboard'), 'success');
```
🤖

File: packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx [hunk 3]
```diff
@@ -72,14 +97,40 @@ function SaveMnemonic({ className, onNextStep, onPreviousStep, seed }: Props): R
...
   );
 }
 
+const StyledCheckbox = styled(Checkbox)`
+  margin: 0;
+  font-size: 14px;
+  letter-spacing: 0.07em;
+  line-height: 145%;
+
+  && .checkbox-ui {
```
Comment (jalooc) [inline packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx:119]: Class referencing of nested components is quite nasty. Any chance we could fix this with changes in `<Checkbox>` itself?

File: packages/extension-ui/src/components/Checkbox.tsx [hunk 4]
```diff
@@ -151,7 +150,7 @@ export default styled(Checkbox)(
 
   &:hover {
     label .checkbox-ui {
-      outline-color:  ${theme.primaryColor};
+      box-shadow: 0 0 0 1px ${theme.primaryColor};
```
Comment (jalooc) [inline packages/extension-ui/src/components/Checkbox.tsx:153]: Good call 👍

File: packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx (hunk 2 above)
Comment (youPickItUp) [reply packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx:42]: I understood this checkbox differently, more like "As a user I've made sure that the secret is saved and secure".
Coping to clipboard isn't enough for me, wdyt?

Comment (jalooc) [reply packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx:42]: Fair enough 👍

**Pull request merged**